indico/modules/events/editing/templates/management/base.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'events/management/base.html' %}
+{% extends 'events/management/full_width_base.html' %}
```

Comment:
Was this really intended? On a wide screen this looks rather terrible...

![image](https://user-images.githubusercontent.com/179599/230486755-8eeecfb2-f1ba-4d24-a743-c73ea7fffadb.png)